server/bundles/io.cloudbeaver.service.auth/schema/service.auth.graphqls

```diff
@@ -158,6 +166,12 @@ extend type Query {
     # If forceSessionsLogout=true then kill another sessions
     authLogin(provider: ID!, configuration: ID, credentials: Object, linkUser: Boolean, forceSessionsLogout: Boolean): AuthInfo!
 
+    @since(version: "25.0.3")
+    asyncAuthLogin(provider: ID!, configuration: ID, linkUser: Boolean, forceSessionsLogout: Boolean): AsyncAuthInfo!
+    @since(version: "25.0.3")
+    asyncAuthTaskResult(taskId: String!): AsyncAuthResult!
```

Comment:
will this method return all user tokens or just new added by this authentication?

Reply:
if it will return all tokens, then we don't need this method; we can use `userInfo`

Reply:
new user tokens, behavior the same as in `authUpdateStatus`